File: Sheet 1/V_Comparison.py
s = input()
res = s.split()
# ['5', '>', '4']
r = "Right"
w = "Wrong"

if res[1] == "=":
    if int(res[0]) == int(res[2]):
        print(r)
    else:
        print(w)
elif res[1] == ">":
    if int(res[0]) > int(res[2]):
        print(r)
    else:
        print(w)
else:
    if int(res[0]) < int(res[2]):
        print(r)
    else:
        print(w)

File: Sheet 1/V/V_Comparison.cpp
#include<bits/stdc++.h>
#define endl '\n'
#define long long int int
using namespace std;

void solve()
{
    int a, b;
    char ch;
    cin >> a >> ch >> b;
    if(ch == '=')
    {
        if(a == b)
            cout << "Right" << endl;
        else
            cout << "Wrong" << endl;
    }
    else if(ch == '<')
    {
        if(a < b)
            cout << "Right" << endl;
        else
            cout << "Wrong" << endl;
    }
    else{
        if(a > b)
            cout << "Right" << endl;
        else
            cout << "Wrong" << endl;
    }
}
int main()
{
    int t;
    // cin >> t;
    t = 1;
    while(t--)
    {
        solve();
    }

    return 0;
}